refresh_rhcos.sh

```diff
@@ -60,21 +59,21 @@ else
 fi
 
 echo "Verifying image..."
-if ! sha256sum $LOCAL_IMAGE_FILE | grep -q $IMAGE_SHA; then
-    echo Downloaded image is corrupted. Exiting...
+if ! shasum -qa256 -c <(echo -n "${IMAGE_SHA}  ${LOCAL_IMAGE_FILE}"); then
```

Comment:
```
zsh: shasum: command not found...
Install package 'perl-Digest-SHA' to provide command 'shasum'? [N/y] N
```
You can also pass the `-c` arg to `sha256sum` if you'd like.

Reply:
I wrote that out of habit, I didn't notice that the script was using a different binary.

Should be fixed, thanks!